src/app/02-form/select/select.component.ts

```diff
@@ -14,12 +14,16 @@ export class SelectComponent implements OnInit {
   themes = [];
 
   selection;
+  optionGroupSelection;
 
   hasCheckbox = false;
 
   isMultiple = false;
 
+  disableOptionGroup2 = false;
```

Comment:
what happen to selected value which parent group become disable ?

Reply:
same thing as when a selected option is disabled which was nothing...

added deselection of disabled option and unit testing